src/inverseRobot.js

```diff
@@ -6,8 +6,25 @@
  * @return {object}
 */
 
-function inverseRobot(robot) {
-  // write code here
+function inverseRobot(oldRobot) {
+  const newRobot = {};
+  let newRobotValue = '';
+  let newRobotKey = '';
```

Comment:
The variable 'newRobotKey' is not necessary, you can directly use 'oldRobot[oldKey]' in the loop.